<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.6.40">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="Sofia Kaiaua1, Deborah Giles2,1, Amy Van Cise1">

<title>Investigating Annual and Seasonal Diet Preference of Southern Resident Killer Whales in the Salish Sea: A Genetic Metabarcoding Approach</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
  margin-bottom: 0em;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>


<script src="SRKW_diet_from_WO_samples_files/libs/clipboard/clipboard.min.js"></script>
<script src="SRKW_diet_from_WO_samples_files/libs/quarto-html/quarto.js"></script>
<script src="SRKW_diet_from_WO_samples_files/libs/quarto-html/popper.min.js"></script>
<script src="SRKW_diet_from_WO_samples_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="SRKW_diet_from_WO_samples_files/libs/quarto-html/anchor.min.js"></script>
<link href="SRKW_diet_from_WO_samples_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="SRKW_diet_from_WO_samples_files/libs/quarto-html/quarto-syntax-highlighting-549806ee2085284f45b00abea8c6df48.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="SRKW_diet_from_WO_samples_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="SRKW_diet_from_WO_samples_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="SRKW_diet_from_WO_samples_files/libs/bootstrap/bootstrap-8a79a254b8e706d3c925cde0a310d4f0.min.css" rel="stylesheet" append-hash="true" id="quarto-bootstrap" data-mode="light">


</head>

<body class="fullcontent">

<div id="quarto-content" class="page-columns page-rows-contents page-layout-article">

<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Investigating Annual and Seasonal Diet Preference of Southern Resident Killer Whales in the Salish Sea: A Genetic Metabarcoding Approach</h1>
</div>



<div class="quarto-title-meta">

    <div>
    <div class="quarto-title-meta-heading">Author</div>
    <div class="quarto-title-meta-contents">
             <p>Sofia Kaiaua<sup>1</sup>, Deborah Giles<sup>2,1</sup>, Amy Van Cise<sup>1</sup> </p>
          </div>
  </div>
    
  
    
  </div>
  


</header>


<ol type="1">
<li>University of Washington, Seattle, WA</li>
<li>Wild Orca, San Juan, WA</li>
</ol>
<p><br></p>
<p>Corresponding author email: avancise@uw.edu</p>
<p><br></p>
<section id="background" class="level3">
<h3 class="anchored" data-anchor-id="background">Background</h3>
<p>The endangered southern resident killer whale (<em>Orcinus orca ater</em>, hereafter SRKW) population has not recovered since the 1970s despite protective measures (NOAA/NMFS, 2016). SRKW exhibit culturally learned behaviors, including diet preferences, primarily consuming Chinook (<em>Oncorhynchus tshawytscha</em>), chum (<em>O. keta</em>), and coho (<em>O. kisutch</em>) salmon <span class="citation" data-cites="Hanson2021 vancise_etal24">(<a href="#ref-Hanson2021" role="doc-biblioref">Hanson et al. 2021</a>; <a href="#ref-vancise_etal24" role="doc-biblioref">Van Cise et al. 2024</a>)</span>. Decadal decline in size and abundance of preferred prey may contribute to malnutrition or starvation in this population <span class="citation" data-cites="Hanson2021">(<a href="#ref-Hanson2021" role="doc-biblioref">Hanson et al. 2021</a>)</span>; additionally, prey limitation can increase their vulnerability to cumulative environmental stressors <span class="citation" data-cites="Wade2012">(<a href="#ref-Wade2012" role="doc-biblioref">Wade, Reeves, and Mesnick 2012</a>)</span>. Comprehensive analysis of seasonal and annual variability in SRKW diet is vital to constructing recovery strategies for this endangered species; genetic metabarcoding of prey DNA from historically collected fecal samples promises to fill existing data gaps, providing vital insight regarding foraging behavior during non-summer months as well as greater resolution into seasonal variability in foraging behavior within pods, which is currently poorly understood <span class="citation" data-cites="vancise_etal24">(<a href="#ref-vancise_etal24" role="doc-biblioref">Van Cise et al. 2024</a>)</span>. In this pilot study, we sequence a set of fecal samples collected by Wild Orca between 2019 and 2024. The primary goals of this pilot study are to explore the feasibility of applying a genetic metabarcoding approach to samples collected and processed using a variety of methods, and to investigate seasonal and annual variability in the prey species identified in this sample set.</p>
</section>
<section id="methods" class="level3">
<h3 class="anchored" data-anchor-id="methods">Methods</h3>
<p>We collaborated with Wild Orca to collect fecal samples from SRKW, and supplemented samples collected during the 2024 field season with historical samples collected between 2019-2023. Samples were either swabbed using a cotton-tipped swab or scooped out of the water using a sterilized scoop. A subset of wet scoop samples were later lyophilized (dried), and some were stored as wet samples. Whole genomic DNA was extracted from swabbed, wet scooped, and lyophilized samples using the QIAmp Fast DNA Stool Mini Kit (Qiagen). We quantified the amount of DNA extracted from each sample, then amplified the 16S mitochondrial region using published primers and protocols (Van Cise et al.&nbsp;2024). We determined PCR success through visual examination using gel electrophoresis, and re-amplified unsuccessful samples under a variety of parameters to increase sample success rate. Those parameters included: 1:10 dilution, 1:2 dilution, DNA cleaned using a Zymo Clean and Concetrate kit, and 2x concetration of cleaned DNA. After all iterations of PCR1 were run, we collected those samples with successfully amplified product to continue through the library preparation protocol.</p>
<p>Next we cleaned the amplified PCR product using the AMPure bead clean protocol and indexed each sample using Unique Dual Indexes (UDI, Illumina, Inc). Because a significant amount of DNA can be lost during each bead cleaning step, we added 8 duplicate samples to the indexing reaction using product that had not been cleaned to determine whether this AMPure bead clean step significantly affects indexing and downstream results. We then performed a second AMPure bead clean to remove all unused reagents, and pooled all samples to contain an equimolar mix of prey DNA from each sample.</p>
<p>Once the amplified product was pooled, we performed a final gel cleaning step using the MinElute PCR purificaton kit (Qiagen, Inc) to remove any non-target DNA fragments, then diluted and denatured the DNA according to the MiSeq sequencing protocol and loaded the pooled DNA and 20% PhiX control onto an in-house MiSeq sequencer.</p>
<p>We quantified the concentration of DNA (ng/ul) in all samples using a Qubit flourometer (ThermoFisher, Inc.) immediately post extraction and post indexing, both in order to guide decision-making regarding the volume of DNA extract to use in the pipeline, and to draw inference regarding the effect of post-amplification cleaning on DNA concentration.</p>
<p>We processed the raw sequence data using a custom dada2-based bioinformatic pipeline (version 1.34.0), implemented in R (version 4.4.2). Briefly, this pipeline removes low quality and short sequence reads before merging sequences and identifying unique individual sequences, called amplicon sequence variants (ASVs), based on sequence similarity. We then compared ASVs to a custom 16S reference database of North Pacific fish species, targeting SRKW prey <span class="citation" data-cites="vancise_etal24">(<a href="#ref-vancise_etal24" role="doc-biblioref">Van Cise et al. 2024</a>)</span>, using a naive Bayesian algorithm to classify each ASV to the lowest possible taxonomic level <span class="citation" data-cites="Wang2007">(<a href="#ref-Wang2007" role="doc-biblioref">Wang et al. 2007</a>)</span>. Following this, we removed all killer whale sequences from the dataset, then removed any samples that only contained killer whale sequences.</p>
<p>We performed all downstream data analysis in R using phyloseq (version 1.50.0), vegan, and ggplot.</p>
</section>
<section id="results" class="level3">
<h3 class="anchored" data-anchor-id="results">Results</h3>
<section id="genetic-sequencing" class="level4">
<h4 class="anchored" data-anchor-id="genetic-sequencing">Genetic sequencing</h4>
<p>Of the 76 SRKW samples provided by Wild Orca, we successfully amplified 37 samples. Most of these (n = 21) were successful in the first iteration of amplification (i.e.&nbsp;original concentration). We gained 1 additional sample in the 1:10 dilution, 2 samples in the 1:2 dilution, 5 samples from the Zymo-cleaned extract, and 8 samples at 2x concentration of cleaned extract. All 38 successfully amplified samples were then cleaned and indexed, and 8 duplicates were indexed without cleaning. This resulted in 45 samples that underwent a post-index bead clean and were pooled into a final product for sequencing.</p>
<p>After aligning and merging forward and reverse raw sequence reads, we retained 36 of the 45 samples; the remaining 9 samples were removed due to low sequence quality. Table 1 shows the mean number of reads retained after each step in the dada2 quality control pipeline; the final number of reads per sample ranged from 458 to 368,431. Of the 9 samples that were not successfully sequenced, 5 were cleaned samples with successful uncleaned duplicates.</p>
<div class="cell">
<div class="cell-output-display">
<table class="caption-top table table-sm table-striped small">
<caption>Table 1. Mean number of reads retained at each step in the dada2 QAQC pipline.</caption>
<colgroup>
<col style="width: 13%">
<col style="width: 15%">
<col style="width: 16%">
<col style="width: 16%">
<col style="width: 12%">
<col style="width: 13%">
</colgroup>
<thead>
<tr class="header">
<th style="text-align: center;">input</th>
<th style="text-align: center;">filtered</th>
<th style="text-align: center;">denoisedF</th>
<th style="text-align: center;">denoisedR</th>
<th style="text-align: center;">merged</th>
<th style="text-align: center;">nonchim</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;">239,011</td>
<td style="text-align: center;">68,439</td>
<td style="text-align: center;">68,400</td>
<td style="text-align: center;">68,381</td>
<td style="text-align: center;">67,791</td>
<td style="text-align: center;">67,259</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>After removing host sequences, we additionally removed WADE-002-076 and WADE-002-077 which did not have any sequence reads assigning to prey species. This resulted in a final total of 34 samples with prey sequence data that were used in downstream data analyses and visualization.</p>
</section>
<section id="identification-of-prey-species" class="level4">
<h4 class="anchored" data-anchor-id="identification-of-prey-species">Identification of prey species</h4>
<p>Chinook salmon was the most common prey species across all samples during all time periods (Figure 1). Coho salmon was the next most common prey species across all samples with proportions up to 28.4% (Table 2). Chum salmon was the third most common prey species across all samples. Lingcod (<em>Ophiodon elongatus</em>) was present in early summer months in proportions up to 5.2%. Uncommonly detected species included arrowtooth flounder (<em>Atheresthes stomias</em>), steelhead (<em>Oncorhynchus mykiss</em>), swordspine rockfish (<em>Sebastes ensifer</em>), Pacific dover sole (<em>Microstomus pacificus</em>), and Alaska polluck (<em>Gadus chalcogrammus</em>), were in low frequencies throughout all months (≤1%). Great white shark (<em>Carcharodon carcharias</em>) represented approximately 20% of one sample collected on Sept 8, 2022 near San Juan Island.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="SRKW_diet_from_WO_samples_files/figure-html/Figure%202-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption>Fig 1. Relative abundance of prey items detected in southern resident killer whale samples. Species are only included if they make up &gt;1% of at least one sample. Each sample shown on x axis, proportion of each species (indicated by color) indicated on the y axis.</figcaption>
</figure>
</div>
</div>
</div>
</section>
<section id="seasonal-and-annual-variability-in-diet" class="level4">
<h4 class="anchored" data-anchor-id="seasonal-and-annual-variability-in-diet">Seasonal and annual variability in diet</h4>
<p>When samples are organized into summer and non-summer months (Figure 2), we see that, in general, diet is more variable in non-summer months than in summer months. Chinook is the most common species in both seasons, and we observed that chum and coho both increased in late summer to early fall months. Notably, we observed lingcod, steelhead, sockeye salmon and arrowtooth flounderin the summer diet, which have not been identified as part of the summer diet in previous studies.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="SRKW_diet_from_WO_samples_files/figure-html/Seasonal%20Diet%20Composition-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption>Fig 2. Seasonal variability in southern resident killer whale diet (top) in summer vs.&nbsp;non-summer months and (bottom) per species-month.</figcaption>
</figure>
</div>
</div>
</div>
<p>We also report mean monthly abundance for each species in table format below:</p>
<div class="cell">
<div class="cell-output-display">
<table class="caption-top table table-sm table-striped small">
<caption>Table 2. Mean proportion of each prey species by month.</caption>
<colgroup>
<col style="width: 5%">
<col style="width: 14%">
<col style="width: 17%">
<col style="width: 13%">
<col style="width: 15%">
<col style="width: 18%">
<col style="width: 14%">
</colgroup>
<thead>
<tr class="header">
<th style="text-align: center;">Month</th>
<th style="text-align: center;">Atheresthes stomias</th>
<th style="text-align: center;">Hippoglossus stenolepis</th>
<th style="text-align: center;">Oncorhynchus keta</th>
<th style="text-align: center;">Oncorhynchus kisutch</th>
<th style="text-align: center;">Oncorhynchus tshawytscha</th>
<th style="text-align: center;">Ophiodon elongatus</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;">5</td>
<td style="text-align: center;">0.01766</td>
<td style="text-align: center;">0</td>
<td style="text-align: center;">0.0002672</td>
<td style="text-align: center;">0.003044</td>
<td style="text-align: center;">0.9239</td>
<td style="text-align: center;">0.04009</td>
</tr>
<tr class="even">
<td style="text-align: center;">6</td>
<td style="text-align: center;">0.009371</td>
<td style="text-align: center;">0</td>
<td style="text-align: center;">0.001937</td>
<td style="text-align: center;">0.0002625</td>
<td style="text-align: center;">0.9263</td>
<td style="text-align: center;">0.05952</td>
</tr>
<tr class="odd">
<td style="text-align: center;">7</td>
<td style="text-align: center;">0</td>
<td style="text-align: center;">0</td>
<td style="text-align: center;">0</td>
<td style="text-align: center;">0.000919</td>
<td style="text-align: center;">0.9947</td>
<td style="text-align: center;">0</td>
</tr>
<tr class="even">
<td style="text-align: center;">8</td>
<td style="text-align: center;">0.007008</td>
<td style="text-align: center;">0.002434</td>
<td style="text-align: center;">0.1975</td>
<td style="text-align: center;">0.2197</td>
<td style="text-align: center;">0.56</td>
<td style="text-align: center;">0</td>
</tr>
<tr class="odd">
<td style="text-align: center;">9</td>
<td style="text-align: center;">0</td>
<td style="text-align: center;">0</td>
<td style="text-align: center;">0.07135</td>
<td style="text-align: center;">0.2839</td>
<td style="text-align: center;">0.6214</td>
<td style="text-align: center;">0</td>
</tr>
<tr class="even">
<td style="text-align: center;">10</td>
<td style="text-align: center;">0</td>
<td style="text-align: center;">0</td>
<td style="text-align: center;">0</td>
<td style="text-align: center;">0</td>
<td style="text-align: center;">1</td>
<td style="text-align: center;">0</td>
</tr>
<tr class="odd">
<td style="text-align: center;">11</td>
<td style="text-align: center;">0</td>
<td style="text-align: center;">0.1868</td>
<td style="text-align: center;">0</td>
<td style="text-align: center;">0</td>
<td style="text-align: center;">0.8132</td>
<td style="text-align: center;">0</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Finally, we observed a large amount of diet variability among years, with no immediately discernible trends in diversity or content (Figure 3).</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="SRKW_diet_from_WO_samples_files/figure-html/Annual%20Diet%20Composition-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption>Fig 3. Annual variability in souther resident killer whale diet composition (top) and diversity (bottom).</figcaption>
</figure>
</div>
</div>
</div>
</section>
</section>
<section id="discussion" class="level3">
<h3 class="anchored" data-anchor-id="discussion">Discussion</h3>
<p>Our findings align with previous studies indicating significant diet contributions from major prey species including Chinook, coho, and chum salmon <span class="citation" data-cites="Hanson2021 vancise_etal24">(<a href="#ref-Hanson2021" role="doc-biblioref">Hanson et al. 2021</a>; <a href="#ref-vancise_etal24" role="doc-biblioref">Van Cise et al. 2024</a>)</span>. Similar to those previous studies, we observed a decrease in Chinook proportions in August and September compared to early summer months, made up for by increasing proportions of chum and coho salmon increase.</p>
<p>This pilot study successfully generated data for 34 samples collected from all three SRKW pods across a seven month time period and spanning five years; because we know that SRKW diet can vary substantially seasonally and among pods, it is important to remember that this small sample size may have biased our results. With that said, we observed several patterns in this initial subset of data that merit follow-up once the full sample set from Wild Orca is sequenced and combined with the existing sequence data from NOAA samples. We mention these observations below.</p>
<p>Notably, our data suggests a greater prey species contribution from coho salmon compared to existing studies identifying chum salmon as the second greatest contributor to SRKW diet. This may be an effect of bias due to small sample size, or may reflect annual fluctuations in the proportion of coho and chum in SRKW diet.</p>
<p>The detection of steelhead and lingcod in the summer diet supplements existing data indicating that these species were only part of the diet in winter months (Jan-Mar; <span class="citation" data-cites="vancise_etal24">Van Cise et al. (<a href="#ref-vancise_etal24" role="doc-biblioref">2024</a>)</span>). Similarly, sockeye salmon have previously been identified in the diet of SRKW in February and September; here we observe sockeye salmon in samples collected in July and November. Arrowtooth, previously observed in SRKW diet primarily in October, was observed in this sample set in samples collected in May, June, and August. These findings expand our understanding of SRKW seasonal foraging ecology, suggesting that a number of non-salmonid species might be consumed opportunistically throughout the year.</p>
<p>The most surprising observed species in this sample set is the great white shark, which is a species that is not known to inhabit the inland waters of the Salish Sea. In order to confirmed this observation we conducted a search of NCBI Genbank, and international database of genetic sequence data, and found that our sequence is a 100% DNA match to existing great white shark species, with no alternative matches. The sample was collected on September 8, 2022 from a large group of SRKW comprising both J and K pod individuals located off San Juan Island, from and individual that was genetically determined to be J40. J pod was near the San Juan Islands on September 6 and 7, and near the Fraser River on September 5. However, K pod was along the outer coast of Washington until the night of September 7, and were acoustically observed to enter the Salish Sea during the night of September 7. Because some matrilines of J pod will sometimes travel with K pod, we don’t know whether J40 was offshore or inland waters in the days before this sample was collected. Similarly, because we don’t know gut transit time for killer whales, we don’t have an accurate estimate for when the shark may have been eaten.</p>
<p>The observation of great white shark in the SRKW diet has a number of implications. SRKW have previously been observed to eat skate <span class="citation" data-cites="Hanson2021 vancise_etal24">(<a href="#ref-Hanson2021" role="doc-biblioref">Hanson et al. 2021</a>; <a href="#ref-vancise_etal24" role="doc-biblioref">Van Cise et al. 2024</a>)</span>, and dogfish sharks <span class="citation" data-cites="Ford2016">(<a href="#ref-Ford2016" role="doc-biblioref">Ford et al. 2016</a>)</span>, but no white shark (i.e.&nbsp;mako sharks, salmon sharks, porbeagle sharks, and great white sharks) has ever been observed in the diet of these animals. We know from morphological observations that offshore killer whales, which are thought to primarily eat sharks, suffer significant wearing and damage to their teeth over their lifetimes. The lack of similar wear in SRKW teeth suggests that shark is a rare prey item - it is possible that this predation event was an anomaly, or that it was precipitated by climate change (northward shifting prey distributions), lack of their regular prey base, or both. On a broader scale, annual or secular trends in diet diversity may reflect shifts in foraging ecology reflecting a shifting and deteriorating prey base; additional sequencing of historical and contemporary samples will be important to testing this hypothesis.</p>
<p>We also detected small amounts of Alaska polluck and swordspine rockfish. Both of these fishes were detected in relatively small proportions, and are known prey of SRKW prey items (e.g.&nbsp;salmonids, flounder, halibut, sablefish), therefore we hypothesize that we are observing these species as secondary prey items (i.e.&nbsp;fishes that were eaten by fishes eaten by killer whales).</p>
<p>Recent studies (Hanson et al.&nbsp;2021, Van Cise et a. 2024) and this pilot study illustrate that genetic metabarcoding of SRKW fecal samples continues to provide new and valuable insight into seasonal variation in diet and foraging ecology. While this is certainly true for winter months, for which we have very few samples, increasing our observational power during summer months has also proven valuable. Our results not only expand our understanding of opportunistic foraging throughout the year in this population, but also add new species to the diet of this population, and may indicate that SRKW are expanding their diet in response to shifts in their prey base.</p>
<p>The sparsity of diet samples over time, space, and social groups makes it so that any new samples enrich our understanding of SRKW foraging ecology in multiple dimensions. Future work by Wild Orca and the San Diego Zoo Wildlife Alliance to sequence historical and contemporary samples, in collaboartion with NOAAs ongoing diet monitoring, promises to provide insight on forgaing ecology that will inform the management of the endangered SRKW population.</p>
</section>
<section id="references" class="level3">
<h3 class="anchored" data-anchor-id="references">References</h3>
<div id="refs" class="references csl-bib-body hanging-indent" data-entry-spacing="0" role="list">
<div id="ref-Ford2016" class="csl-entry" role="listitem">
Ford, Michael J., Jennifer Hempelmann, M. Bradley Hanson, Katherine L. Ayres, Robin W. Baird, Candice K. Emmons, Jessica I. Lundin, Gregory S. Schorr, Samuel K. Wasser, and Linda K. Park. 2016. <span>“Estimation of a Killer Whale (<em>Orcinus Orca</em>) Population’s Diet Using Sequencing Analysis of DNA from Feces.”</span> <em>PLoS ONE</em> 11 (1): 1–14. <a href="https://doi.org/10.1371/journal.pone.0144956">https://doi.org/10.1371/journal.pone.0144956</a>.
</div>
<div id="ref-Hanson2021" class="csl-entry" role="listitem">
Hanson, M Bradley, Candice K Emmons, Michael J Ford, Meredith Everett, Kim Parsons, Linda K Park, Jennifer Hempelmann, et al. 2021. <span>“Endangered Predators and Endangered Prey: Seasonal Diet of Southern Resident Killer Whales.”</span> Edited by David Hyrenbach. <em>PLOS ONE</em> 16 (3): e0247031. <a href="https://doi.org/10.1371/journal.pone.0247031">https://doi.org/10.1371/journal.pone.0247031</a>.
</div>
<div id="ref-vancise_etal24" class="csl-entry" role="listitem">
Van Cise, Amy M., M. Bradley Hanson, Candice Emmons, Dan Olsen, Craig O. Matkin, Abigail H. Wells, and Kim M. Parsons. 2024. <span>“Spatial and Seasonal Foraging Patterns Drive Diet Differences Among North Pacific Resident Killer Whale Populations.”</span> <em>Royal Society Open Science</em> 11 (9): rsos240445. <a href="https://doi.org/10.1098/rsos.240445">https://doi.org/10.1098/rsos.240445</a>.
</div>
<div id="ref-Wade2012" class="csl-entry" role="listitem">
Wade, Paul R., Randall R. Reeves, and Sarah L. Mesnick. 2012. <span>“Social and Behavioural Factors in Cetacean Responses to Overexploitation: Are Odontocetes Less <span>‘Resilient’</span> Than Mysticetes?”</span> <em>Journal of Marine Biology</em> 2012: 1–15. <a href="https://doi.org/10.1155/2012/567276">https://doi.org/10.1155/2012/567276</a>.
</div>
<div id="ref-Wang2007" class="csl-entry" role="listitem">
Wang, Qiong, George M. Garrity, James M. Tiedje, and James R. Cole. 2007. <span>“Naive Bayesian Classifier for Rapid Assignment of rRNA Sequences into the New Bacterial Taxonomy.”</span> <em>Applied and Environmental Microbiology</em> 73 (16): 5261–67. <a href="https://doi.org/10.1128/AEM.00062-07">https://doi.org/10.1128/AEM.00062-07</a>.
</div>
</div>
</section>
<section id="appendix-i-protocol-optimization" class="level3">
<h3 class="anchored" data-anchor-id="appendix-i-protocol-optimization">Appendix I: Protocol Optimization</h3>
<p>Because sample collection and processing differs between Wild Orca and NOAA, it was neccessary to optimize the lab protocol used to generate genetic sequence data. The following tests were conducted:</p>
<ol type="1">
<li>Analysis of extracted DNA concentration by sample type: wet scoop, swab, or lyophilized (dried).</li>
<li>Analysis of the difference in extracted DNA concentration before and after cleaning using a Zymo Clean and Concentrate kit (Zymo, Inc.).</li>
<li>Analysis of PCR success by DNA concentration.</li>
<li>Analysis of the effect of a post-amplification cleaning step on final DNA concentration, sequencing depth (read count), and resulting analysis of prey species in each sample.</li>
</ol>
<section id="results-1" class="level4">
<h4 class="anchored" data-anchor-id="results-1">Results</h4>
<p>Results of each of these analyses are shown below, and in a poster presented at the UW Summer Research Symposium in August 2024.</p>
<p>Based on these results, the WADE Lab genetic metabarcoding protocol was optimized for WO samples in the following way:</p>
<ol type="1">
<li>Ideal sample input is 0.1g of lyophilized (dried) sample material. Fecal samples collected using a plastic scoop should be passed through a net or sieve after centrifugation to remove seawater, and/or lyophilized to remove water content. Swab samples had a high rate of failure due to very low sample volume.</li>
<li>All samples failing the amplification step should be cleaned and re-amplified using 2x input volume (up to 10 ul).</li>
<li>Samples that still fail the amplification step may be diluted to a DNA concentration of 12-45 ng/ul to increase likelihood of amplification success.</li>
<li>Post amplification cleaning step should be removed, retaining the cleaning step after indexing and the final gel cleaning for size selection.</li>
<li>Samples with a final DNA concentration too low for detection should not be pooled for sequencing.</li>
</ol>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="SRKW_diet_from_WO_samples_files/figure-html/protocol%20optimization-1.png" class="img-fluid figure-img" width="1536"></p>
<figcaption>Supplemental Figure 1. Top: Extracted DNA concentration by sample type (left) and before and after cleaning (middle), and amplification success by DNA concentration. Bottom: Comparison among duplicated samples with and without a post-amplification cleanup step. Left: DNA concentration (ng/ul) after the post-index cleanup step. Middle: Final read count post-QAQC pipeline. Right: Proportional abundance of prey by species.</figcaption>
</figure>
</div>
</div>
</div>
<p>Supplemental Figure 1 (top left) shows the difference in extracted DNA concentration across each of three difference sample types collected by Wild Orca. Wet scoops are collected from the sea surface using a sterilized plastic scoop and centrifuged before pouring off as much seawater as possible. Extracted DNA concentrations from this type sample were highly variable, likely due to variation in seawater content. Swabbed samples are a small volume of fecal matter smeared onto a cotton tip, and resulted in low DNA concentrations and a high rate of sample failure. Lyophilized (dried) samples had the highest consistency in producing ideal DNA concentration.</p>
<p>Supplemental Figure 1 (top) also shows the difference in extracted DNA concentration before and after cleaning (middle), indicating a high rate of DNA loss, and the amplification success rate by extracted DNA concentration (right), indicating that samples with ~12-45 ng/ul of extracted DNA had the highest rate of success.</p>
<p>Supplemental Figure 1 (bottom) shows the difference in final DNA concentration (left), read count (middle), and proportional abundance (right) between duplicated samples with and without a post-amplification cleaning step. Although highly variable, post-amplification cleaning on average resulted in a loss of DNA. In 5 of the 8 duplicates, DNA concentration was reduced to near 0 ng/ul (below Qubit detection levels), resulting in a failure to sequence these samples. In the three samples for which both duplicates were successfully sequenced, the proportional abundance of prey species was identical between duplicates.</p>
<p>Sofia Kaiaua presented the intermediate results of this research at the UW Summer Research Symposium following a 10-week internship program. This presentation is included as Supplemental Figure 2.</p>
<div id="id" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="Data/Kaiaua.WO_labwork_optimization_poster_2024.png" class="class img-fluid figure-img" data-align="center"></p>
<figcaption>Supplemental Figure 2. Poster presented by Sofia Kaiaua at the UW Summer Research Symposium in August 2024, detailing intermediate results of genetic metabarcoding protocol optimization.</figcaption>
</figure>
</div>
</section>
</section>

</main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->




</body></html>